test/unit/module/real/core/rem.cpp

```diff
@@ -73,9 +73,9 @@ EVE_TEST( "Check behavior of rem on wide"
   using eve::rem;
   using eve::pedantic;
   using eve::detail::map;
-  TTS_ULP_EQUAL( pedantic(rem)(a0, a1), map([](auto e, auto f) { return eve::rem(e, f); }, a0, a1), 30);//fma not avail scalar double it seems
+  TTS_ULP_EQUAL( pedantic(rem)(a0, a1), map([](auto e, auto f) { return eve::rem(e, f); }, a0, a1), 32);//fma not avail scalar double it seems
```

Comment:
Why is this changing?

Reply:
as the size of the vector changed to includes the new aggregate size for avx512, it appears that one value actually trigger a slightly larger ULP.